### PR TITLE
It's just better

### PR DESCRIPTION
**Pull request by unimariJo**
You can add new domains without deleting certificates of the old, bug fixes, code formatting.

### CODE REVIEW

**Comment by wmnnd** [comment]
Thank you for this very extensive pull request! I like the changes you’ve added but I would ask you not to rename `init-letsencrypt.sh` to `cerbot.sh`. If you change this back, I’d be happy to do a final review and merge your PR.

**Comment by unimariJo** [comment]
I renamed `certbot.sh` to `init-letsencryp.sht` and made some other improvements & fixes.

File: init-letsencrypt.sh [hunk 1]
```diff
@@ -6,6 +6,11 @@ data_path="./data/certbot"
 email="" # Adding a valid address is strongly recommended
 staging=0 # Set to 1 if you're testing your setup to avoid hitting request limits
 
+if [ "$EUID" -ne 0 ]; then
```
Comment (wmnnd) [inline init-letsencrypt.sh]: Why should this be run as root? I don’t think the script should require root privileges.

Comment (unimariJo) [reply init-letsencrypt.sh]: I made root privileges optional.

**Comment by wmnnd** [comment]
I’m merging it in the dev branch so I can give it another try before merging it back in master :)

**Pull request merged**